Gemfile.lock

```diff
@@ -1,3 +1,12 @@
+GIT
+  remote: https://github.com/d12/graphql-client.git
```

Comment:
Why are we using your fork instead of `GitHub/graphql-client`?

Reply:
I touched on that in the Gemfile :)

https://github.com/education/classroom/pull/1635/files#diff-8b7db4d5cc4b8f6dc8feb7030baa2478R31

Reply:
> Using fork of graphql-client which fixes a bug in graphql-client on Rails 5